nucleus/common/mbeanserver/src/main/java/org/glassfish/admin/mbeanserver/RMIConnectorStarter.java

```diff
@@ -137,7 +139,7 @@ public RMIConnectorStarter(
         }
 
         final boolean ENABLED = true;
-        mBindToSingleIP = ENABLED && !(address.equals("0.0.0.0") || address.equals(""));
+        mBindToSingleIP = ENABLED && !(address.equals("0.0.0.0") || address.equals("*")|| address.equals("")); 
```

Comment:
This line seems like it should work, but for whatever reason when it should be binding multiple IPs it still doesn't. Possibly due to the logic on line 160.